README.md

```diff
@@ -28,24 +28,28 @@ The development server will proxy some requests to either of these and must be p
 
 To use the development server with a local middleware and collector backend run:
 ```
-CONCORDIUM_MIDDLEWARE_URL='http://localhost:8081' CONCORDIUM_COLLECTOR_BACKEND_URL='http://localhost:12000' npm run dev
+STATS_VERSION=3 CONCORDIUM_MIDDLEWARE_URL='http://localhost:8081' CONCORDIUM_COLLECTOR_BACKEND_URL='http://localhost:12000' npm run dev
```

Comment:
If `STATS_VERSION` defaulted to the empty string then it would not filter out any versions. So I think we could reasonably omit it in these commands.

Reply:
I deliberately did not want it to default to anything, since it is then easy to forget if it does.